checker/checkers.go

```diff
@@ -51,14 +53,23 @@ func (c *creatorStruct) Priority() okgo.CheckerPriority {
 	return c.priority
 }
 
+func (c *creatorStruct) MultiCPU() okgo.CheckerMultiCPU {
+	return c.multiCPU
+}
+
 func (c *creatorStruct) Creator() CreatorFunction {
 	return c.creator
 }
 
-func NewCreator(checkerType okgo.CheckerType, priority okgo.CheckerPriority, creatorFn CreatorFunction) Creator {
+func NewCreator(
+	checkerType okgo.CheckerType,
+	priority okgo.CheckerPriority,
+	multiCPU okgo.CheckerMultiCPU,
+	creatorFn CreatorFunction) Creator {
```

Comment:
Given that we expect most checks to use the default value for this, I think we'll have a smoother upgrade path if we don't introduce this compile break -- as currently written, every check that updates its `okgo` dependency to this new version will have a compile break that we'll need to fix by hand.

As such, I think it would be easier on us if we keep the `NewCreator` function signature as-is, and just add a `NewCreatorWithMultiCPU` or similar function and have the old `NewCreator` function call into that one with a default `MultiCPU` value of `false`. That way, we'll only need to proactively update the checks where we want to set the value to `true`.

Reply:
Yep makes sense. Fixed!